### PR TITLE
Webpack 3, Remove Gulp

### PR DESCRIPTION
**Pull request by trainerbill**
Same as cross-domain-utils.  I don't know what flow-runtime is but it is not working in this project and I found out that .babelrc needs to be complete configuration for each env so I will have to update cross-domain-utils as well.

### CODE REVIEW

**Comment by trainerbill** [comment]
Fixed flow-runtime.  Ready for review and merge

**Comment by trainerbill** [comment]
@bluepnume

File: package.json [hunk 1]
```diff
@@ -2,9 +2,25 @@
   "name": "cross-domain-safe-weakmap",
   "version": "1.0.10",
   "description": "Cross domain safe WeakMap.",
-  "main": "index.js",
+  "main": "dist/cross-domain-safe-weakmap.js",
+  "module": "dist/module",
   "scripts": {
-    "test": "gulp test"
+    "build": "npm run babel & npm run webpack && npm run flow:build",
```
Comment (bluepnume) [inline package.json]: Let's run test as a prerequisite for build

Comment (bluepnume) [inline package.json]: Also using ` & ` doesn't seem to work very well for me -- kicks the task into the background, so it's not great if we're using these npm commands in any shell scripts that need to wait for the return codes.

Comment (trainerbill) [inline package.json]: So what do you want to do?  I don't really know why you have the shell scripts anyway...  For publishing this is what I do:

https://github.com/trainerbill/hapi-paypal/blob/master/package.json#L28-L30

I don't see much of a reason to have your publish script.  I would add a git checkout dist to make sure you never commit dist on a release and then have a release:dist that does everything without checking out dist.

Comment (trainerbill) [inline package.json]: Also I don't think a test before building is a good idea...  What if you just want to know if it builds correctly without going through a million tests?

Comment (bluepnume) [inline package.json]: The publish.sh script does a bunch of stuff that it wouldn't be easy to compress into a single line. Seems like having `package.json` point to shell scripts should be a pretty standard thing, right? Writing a lot of code directly in `package.json` feels like a bit of a mess.

> Also I don't think a test before building is a good idea... What if you just want to know if it builds correctly without going through a million tests?

Then we can just run `run run webpack` right? Typically a build job doesn't just combine the files, it runs tests and lint and everything else that's needed too. I think that's a pretty well established convention.

**Pull request closed without merging**